### DISCUSSION TITLE
Remember last used asset path per entity

### DISCUSSION THREAD

**Discussion by nitech** [craftcms/cms#8977] [Ideas]
### Description
It would make sense if each field would remember the last used asset path. 

Say if you're adding product photos to a PRODUCT entity, and you come back after a few days to add more photos. You'd expect the default path to be the one you used to add the first photos. 

Together with https://github.com/craftcms/cms/issues/7156 this feature would make the authoring process more friendly.

**Comment by brandonkelly** [comment]
That’s actually how it used to work until Craft 3.4, but we changed it per #2778. If this gets some traction we could consider adding a setting for it, but not sure where the best place to put it would be (Assets field setting? User preference? System-wide general config setting?).

**Comment by nitech** [comment]
I suggest a system-wide general config setting. But as I mentioned, it is just a first step in enhancing the authoring experience. #7156 is one of the next steps. 

I understand you've had a focus on accessibility in the authoring experience. This issue and #7156 is not directly accessibility but more usability/ux. Hope to see some traction in that area. 

As a fellow software developer, I know great software take time. Hope your financial model and customer base allows you to hire more skilled developers.
